client/config.ts

```diff
@@ -3,6 +3,7 @@ const debug = process.env.DEBUG
 const config = {
   debug,
   embedderContract: process.env.EMBEDDER_CONTRACT ?? (debug ? '0x9BC52FBcCcde8cEADAEde51a25dBeD489b201e53' : '0x476e14D956dca898C33262aecC81407242f8431A'),
+  embedPlatform: process.env.EMBED_PLATFORM ?? 'notion',
```

Comment:
use ewsType value read from contract, instead of configuring as fixed value at client

Reply:
@polymorpher yes we can get ewsType for registered domain.
My question is how we can get ewsType in /manage page before registration?
Also we need to tell the smart contract that the domain is for whether notion or substack when calling `client.update`

Reply:
Manage page is for setting ewsType of a subdomain config, not getting it. client.update already has the interface for it `update: (sld: string, subdomain: string, ewsType: EWSType, page: string, pages: string[], landingPageOnly: boolean) => Promise<ContractTransaction>`

Reply:
I see.
My question is how the frontend distinguish what manage page (notion vs substack) it will render before the domain registration. Setting ewsType is done inside Manage component, but don't know ewsType at the time of deciding what manage page to render. I could miss something

Reply:
We could provide a selection button where the user can decide which platform they want the subdomain to connect with (and we can render the corresponding manage component based on user selection). A separate, textbox-based input UI is also implemented, outside this repository.

Reply:
I see